static/src/javascripts/projects/common/modules/discussion/comment-count.js

```diff
@@ -68,10 +72,15 @@ define([
                 var format,
                     $node = bonzo(node),
                     url = $node.attr('data-discussion-url') || getContentUrl(node),
+                    shortUrl = $node.attr('data-loyalty-short-url').replace('/p/','') || '',
                     $container,
                     meta,
                     html;
 
+                if (shortUrl && ab.isInVariant('ParticipationHideHalfOfComments', 'hide-comments') &&  CommentBlocker.hideComments(shortUrl) ) {
```

Comment:
Ah ok. That makes sense yeah, thanks for explaining, that's fine then as we're not skewing any data :)